lib/core/src/Cardano/Wallet/Api/Link.hs

```diff
@@ -919,6 +919,13 @@ patchSharedWallet w cred =
   where
     wid = w ^. typed @(ApiT WalletId)
 
+-- Node
```

Comment:
I don't understand this comment. What purpose does it serve?

Reply:
it starts the links to the client of the end-points in the Node group in the API docs

Reply:
```suggestion
--
-- Node
--

```
?